README.md

```diff
@@ -171,12 +172,10 @@ or later, which officially supported C++11.
 3. HuC firmware is needed for AVC/HEVC/VP9 low power encoding bitrate control, including CBR, VBR, etc. As of now, HuC firmware support is disabled in Linux kernels by default. Please, refer to i915 kernel mode driver documentation to learn how to enable it. Mind that HuC firmware support presents in the following kernels for the specified platforms:
    * APL/KBL: starting from kernel 4.11
    * CFL: starting from kernel 4.15
+   * ICL: starting from kernel 5.2
```

Comment:
As of now FW for these platforms did not appear even in the latest 5.4.2: EHL, TGL:
https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tree/drivers/gpu/drm/i915/gt/uc/intel_uc_fw.c?h=v5.4.2#n42

But it did appear on in drm-tip:
https://cgit.freedesktop.org/drm-tip/tree/drivers/gpu/drm/i915/gt/uc/intel_uc_fw.c#n46

Hence I suggest to add this to the readme:
> * EHL, TGL: [drm-tip](https://cgit.freedesktop.org/drm-tip), missed in <= 5.4.2 kernels

Reply:
agree